welkin/models/chat.py

```diff
@@ -0,0 +1,70 @@
+from welkin.models.base import Collection, Resource
+from welkin.pagination import MetaIterator
+
+
+class Chat(Resource):
+    def create(self):
+        return super().post(
+            f"{self._client.instance}/patients/{self._parent.id}/chat/inbound"
+        )
+
+    def __str__(self):
+        return f"{self.sender['clientType']} {self.message}"
+
+
+class Chats(Collection):
+    resource = Chat
+    iterator = MetaIterator
+
+    def get(
+        self,
+        pageSize: int = 20,
```

Comment:
These arguments don't follow [PEP8 conventions](https://peps.python.org/pep-0008/#function-and-method-arguments) (should be snake_case).

The only time we should be using lowerCamelCase is when we're interacting with data that was parsed from a request (like `Encounter().patientId`). If it's purely python functionality, snake_case is correct (like `Encounter().get(patient_id="")`.